.github/workflows/main.yaml

```diff
@@ -50,29 +50,35 @@ jobs:
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
 
-  # run-jax-tests:
-  #   runs-on: ubuntu-latest
-  #   strategy:
-  #     fail-fast: false
-  #     matrix:
-  #       python-version: ["3.9", "3.10", "3.11"]
+  run-jax-tests:
```

Comment:
I guess this is irrelevant with the new structure and all that is needed is to make
```yaml
          extra-specs: |
            python=${{ matrix.python-version }}
            jax
```
os-dependent?

Reply:
Yes, that's much better.